src/backend/distributed/commands/utility_hook.c

```diff
@@ -166,9 +186,11 @@ static bool IsCommandToCreateOrDropMainDB(Node *parsetree);
 static void RunPreprocessMainDBCommand(Node *parsetree);
 static void RunPostprocessMainDBCommand(Node *parsetree);
 static bool IsStatementSupportedFromNonMainDb(Node *parsetree);
-static bool StatementRequiresMarkDistributedFromNonMainDb(Node *parsetree);
-static void MarkObjectDistributedFromNonMainDb(Node *parsetree);
-static MarkObjectDistributedParams GetMarkObjectDistributedParams(Node *parsetree);
+static bool StatementRequiresMarkDistributedGloballyFromNonMainDb(Node *parsetree);
+static bool StatementRequiresUnmarkDistributedLocallyFromNonMainDb(Node *parsetree);
+static void MarkObjectDistributedGloballyFromNonMainDb(Node *parsetree);
+static List * GetMarkObjectDistributedParams(Node *parsetree);
+static void UnMarkObjectDistributedLocallyFromNonMainDb(List *unmarkDistributedList);
```

Comment:
```suggestion
static void UnMarkObjectDistributedLocallyFromNonMainDb(List *unmarkDistributedList);
static List * GetMarkObjectDistributedParams(Node *parsetree);
```

Implementation of GetMarkObjectDistributedParams is provided after UnMarkObjectDistributedLocallyFromNonMainDb, so let's fix the declaration order too.

Reply:
Let's also rename GetMarkObjectDistributedParams to GetDistObjectOperationParams to align its naming with other functions in this code-path.

Also need to rename MarkObjectDistributedParams to DistObjectOperationParams.

Reply:
Done